### PR TITLE
Setup DB with a sample user table

### PR DESCRIPTION
**Pull request by JoshL579**

### CODE REVIEW

File: server/.env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-TEAM_NAME="Shums,Tony Fang, Josh Liu"
-FLASK_ENV="development"
+TEAM_NAME="Shums,Tony,Josh"
```
Comment (sina-jamshidi) [inline server/.env]: we can remove TEAM_NAME and other boilerplate code.
Also make sure your `.env` is not being pushed to git

File: server/config.py [hunk 2]
```diff
@@ -1,3 +1,8 @@
 import os
+from flask_sqlalchemy import SQLAlchemy
 
-TEAM_NAME = os.environ['TEAM_NAME']
+TEAM_NAME = os.environ.get('TEAM_NAME')
+
+SQLALCHEMY_DATABASE_URI = "postgresql://postgres:123456@localhost:5432/postgres"
```
Comment (sina-jamshidi) [inline server/config.py]: this should be in `.env`

**Review by sina-jamshidi — APPROVED** [review]
just a couple of comments on `.env`. When you fix those you can merge!

File: server/.env (hunk 1 above)
Comment (JoshL579) [reply server/.env]: Do you mean that we are not gonna use TEAM_NAME in the future and delete them in both `.env` and `config.py`?

Comment (sina-jamshidi) [reply server/.env]: yup!

**Pull request merged**